test/entities.go

```diff
@@ -146,6 +149,7 @@ func (g *Blocks) New() *flow.Block {
 	return &flow.Block{
```

Comment:
It would be good to actually add a test that checks the signature presence.

Reply:
I added the signature struct on test/entity.go and assertion on client/client.test.go Is that what you mean?

Reply:
Yes perfect, although I don't see the client test, did you push it?

Reply:
Sorry. My bad. I think client_test.go does not need to be updated.

Reply:
Good, then I would consider this done and approved. Thank you